CommonHeader.h

```diff
@@ -0,0 +1,20 @@
+#include <string.h>
```

Comment:
This filename 'commonheader' doesn't convey its contents. Not being specific, it has collected both production and test (stub) function signatures.

Separate the code that doesn't participate in production (end-user) flows